usaspending_api/api_docs/markdown/using_the_api.md

```diff
@@ -340,6 +340,31 @@ The response has three functional parts:
   * `req` - The special code corresponding to this POST request. See [Post request preservation]("#post-requests-preservation") for how to use this
   * `results` - An array of objects corresponding to the data returned by the specified endpoint. Will _always_ be an array, even if the number of results is only one.
 
+### CSV Bulk Downloads
+
+Bulk CSV downloads are available via the `/api/v1/download/` endpoint. This supports any data endpoint via POST or GET. Because these CSV files can take some time to generate, the response of this endpoint contains the status of file generation and the location of the file once it is complete.
+
+Examples
+* To get a CSV of all Awards, you would access `/api/v1/download/awards/`
+* To get a CSV of all Awards with a particular set of filters, you can pass the request checksum, `/api/v1/download/awards/?req=CHECKSUM` or POST the request object to the endpoint
+* To check the status of a CSV download, make note of the `request_checksum` and return to the url, with `req=CHECKSUM` attached. For example, `/api/v1/download/awards/?req=CHECKSUM`
+
+Response
+
+```
+{
+  "location": s3://someURL,
+  "status": "This file has been requested, and is awaiting queueing",
```

Comment:
Former, descriptions I came up with. We can change them whenever. We can check SQS console to see all requests that are waiting.